### ISSUE TITLE
Display "Click to interact" text when onFocusOnly is enabled

### ISSUE THREAD

**Issue by paul121** [farmOS/farmOS-map#93]
@jgaehring in https://github.com/farmOS/farmOS-map/issues/62#issuecomment-589428916

> I wonder if it might also be helpful to have some sort of indicator which pops up on the map on mouse enter when it's inactive. Perhaps just some animated text that slides up and says "Click to interact" or something like that.

This would be a nice addition. Perhaps it could be configurable as well.

**Comment by mstenta** [comment]
Google Maps JS API does something similar when you try to scroll on it: https://developers.google.com/maps/documentation/javascript/overview

I like the way that looks - grays out the whole window with some white text in the center.

**Comment by jgaehring** [comment]
> Perhaps it could be configurable as well.

I wonder if this could be done as a behavior, or something similar. As I mentioned in #62, there are applications like Field Kit that really won't need this at all, so it would be nice if it could be added in the application code rather than in this library.

**Comment by mstenta** [comment]
+1 to the behavior idea!

**Comment by paul121** [comment]
Yes! It could start as behavior in Drupal farmOS and later move to this library as an internal behavior it that becomes needed elsewhere.
